.github/ISSUE_TEMPLATE/request-for-amplification-template.yml

```diff
@@ -52,32 +77,71 @@ body:
         - "#WordCamp"
     validations:
       required: true
+  - type: dropdown
+    id: make-team
+    attributes:
+      label: MakeWP Team
+      description: What MakeWP Team is this request for?  (Select all that apply)
+      multiple: true
+      options:
+        - Accessibility Team
+        - Community Team
+        - Core Team
+        - Design Team
+        - Documentation Team
+        - Hosting Team
+        - Marketing Team
+        - Meta Team
+        - Plugins Team
+        - Polyglots Team
+        - Support Team
+        - Sustainability Team
+        - Themes Team
+        - Training Team
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: summary
+    attributes:
+      label: What do you want to amplify?
+      description: Please provide a short, one-sentence summary of your request.
+    validations:
+      required: true
   - type: textarea
     id: content
     attributes:
-      label: Suggested content
-      description: Using the [WordPress Brand Writing Style Guide](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/), craft your suggested content for this post. The team will review and edit as-needed prior to posting. If you would like additional help, please consult the [#marketing Slack](https://wordpress.slack.com/archives/C0GKJ7TFA).
+      label: Amplification request details
+      description: Please describe what you are looking to amplify or promote. Include as many details as possible. If you have specific content that you would like to use, please note that here. If you would like additional help, please consult the [#marketing Slack](https://wordpress.slack.com/archives/C0GKJ7TFA).
     validations:
       required: true
-  - type: checkboxes
-    id: onbrand
+  - type: input
+    id: links
     attributes:
-      label: Style confirmation
-      options:
-        - label: I've read the WordPress Brand Writing Style Guide
-          required: true
+      label: Links
+      description: Are there any posts, articles, resources or other links that should be shared with this content? Please provide the link and any context.
+      placeholder: ex. "https://wordpress.org/news - This is a link to the page we want people to go to"
+    validations:
+      required: false
   - type: input
-    id: link
+    id: deadline
     attributes:
-      label: Link
-      description: Do you have an article or resource that should be shared with this content?
-      placeholder: ex. https://wordpress.org/news
+      label: Deadline
+      description: Is there a firm deadline for this request? Posts to social platforms are scheduled at least one week in advance. For urgent requests, please post in the [#marketing Slack](https://wordpress.slack.com/archives/C0GKJ7TFA).
+      placeholder: ex. 24 May 2024
```

Comment:
Google said 🤣 

> The correct abbreviation for “for example” is e.g., not “ex.” “Ex” is sometimes used in informal language, but it's best to avoid it, since this abbreviation can be confused with other uses of “ex,” such as its use to mean “former” in a term like “ex-boyfriend.”

Can we use `E.x.` or `E.g.` here?

Reply:
absolutely